### DISCUSSION TITLE
storybook errors

### DISCUSSION THREAD

**Discussion by mathisloge** [vmware/clarity#6655] [Q&A] [answered]
I'm getting a lot of errors in the storybook and can't view any page: 
e.g. https://clarity.design/storybook/core/?path=/story/documentation-angular--page looks like this:
![image](https://user-images.githubusercontent.com/13556116/153017425-cea2421f-1230-49d7-acdf-d1e09544e60f.png)

**Comment by mathisscott** [comment, marked as answer]
We reverted to the website to the Jan 24 release.

Documentation should be available again. We are looking into what happened between now and yesterday's release to cause this breakage on the website.

**Reply by mathisscott** [reply]
I'm sorry for the inconvenience and confusion. 😞

**Reply by mathisloge** [reply]
no worries at all! Thanks for the working site 👍
